<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UICommand</class>
 <widget class="pDockWidget" name="UICommand">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/MessageBox.qrc">
    <normaloff>:/icons/tablog.png</normaloff>:/icons/tablog.png</iconset>
  </property>
  <property name="windowTitle">
   <string>Commands</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="teLog">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>212</red>
            <green>208</green>
            <blue>200</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="tabStopWidth">
       <number>20</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pDockWidget</class>
   <extends>QDockWidget</extends>
   <header location="global">pDockWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/MessageBox.qrc"/>
 </resources>
 <connections/>
</ui>
